apps/asap-server/src/controllers/teams.ts

```diff
@@ -19,6 +19,9 @@ const priorities: Record<TeamRole, number> = {
   'Project Manager': 3,
   Collaborator: 4,
   'Key Personnel': 5,
+  Advisor: 6,
+  Guest: 7,
+  Staff: 8,
```

Comment:
Took a guess at this

Reply:
I works, it's not 100% accurate but we can adjust in the future 🙂.